- review notes from wk3 m3 forward

# What was it About?
Genomic precision medicine by USF is a course about the terminology, basic understanding of genomics. They begin by describing that genomics is the study of the genome and the genome is a collection of introns and exons. Exons contain genes, in humans their is about 20,000 genes. Genes are the coding regions, meaning they code for proteins, the things that do useful work in the cell.

Introns are the space between genes. They aren't entirely useless, they contain promoter regions--these regions help proteins act on specific genes. They also contain information about splicing which affects crossing over during chromosome duplication.

These coding regions (and other regions) can have variation. This variation can cause disease. The types of variation is usually SNPs, but they can also be indels (insertions or deletions), or they can be CNV (copy number variations). This refers to duplications of entire genes. Although you could also have repeats of certain DNA sequences. For example, fragile x is a disease caused by the overduplication of CG repeats, if this happens a few hundred times this causes Fragile X
syndrome.

We can use our models of genomics to understand, predict, and model disease. Especially mendelian disease. Many diseases can be predicted based on Mendel's models.

This is why we offer genetic screens for newborns and children or carrier testing for prospective parents.

There are a few ways of detecting variants that depend on the available sequencing methods. There is whole genome sequencing methods and there is direct targeting sequencing. Karyotyping is one kind of whole genome analysis that is low resolution. You can only see defects in the chromatids themselves.

Cytogenomic arrays are another way of inspecting the whole genome via mass sequencing. The reads are short with NGS so we need to use a reference to help us as a guide in stiching together the pieces. 

There is also a term for whole exome sequencing, this is where you sequence everything that is in the coding regions of DNA. This is only 2%! So this can be advantageous but it's tricky doing this kind of targeted sequencing. Cheaper to analyze 2% of the genome than the whole thing! 85% of mendelian disease happen in the exome and we don't know how to interpret the introme in any case.

One issue with NGS is it is hard to read long repeating strands. This is one difficulty in studying limb regeneration for the Aexocoatl--the genome is 3x bigger than the human genome with huge repetive sequences. This is difficult because sequencing methods break the genetic sequence into small fragments and then align
them together like a giant puzzle.

How do you know what genes cause disease or are risk factors for disease? It's difficult because while many diseases are mendelian and have a purely genetic basis, many diseases are a complex disease that have to interact with the environment, you can have the same genes as a family member but not express the disease just be a carrier depending on your enviroment.

There are many variants that differ from the reference on the order of 5-10 million! 40-100k are in the exome, of those 10-12k are synomyous--no protein change. 8-11k SNPs are non-synonymous in 4-5k genes. There are 200-500k indels and 150 inframe indels in exons, 200-250 cause frameshift mutations. 500-1000 CNVs > 1000 bp in length.

There is difficulty here because now you need to interpret these variants but we can't do that without long term studies! Or a deterministic model (which we don't have).

There are a few diffeernt classifications of variants. Probably damaging mutation types:
- stop-loss
- stop-gained
- frameshift
- splice disruptor

Possibly damaging mutation types are:
- non-synonymous variations
- in frame indel

Likely ok:
- 5'/3' UTR
- synonymous
- intergentic ie. intronic
- non-coding genes

In general, if something is evolutionarily conserved, especially accross species, than it is likely a useful bit, so a variant there is likely deleterious. We can use in vitro (cell based) or in vivo (animal) assays to make basic determinations about whether something is wrong. Then go from there looking for genetic problems.

Associations with diseases can be looked up in a few databases:
- Online Mendelian Inheritance in Man (OMIM)
- clinvar by the NIH

and many more!

We can identify genes via genome wide association studies. Basically we track a large population that have the disease and fine common genes that occur in high frequency. Still, it is difficult to interpret these diseases.

Pharmacokinetics and pharmacodynamics are how drug molecules interact with genomics.

# What I Learned?

# What Could Have Been better?

